package.json

```diff
@@ -6,6 +6,7 @@
     "deploy": "npx hardhat deploy",
     "lint": "yarn eslint && yarn solhint",
     "eslint:fix": "eslint . --ext .ts --fix",
+    "format": "prettier-eslint --write 'test/**/*.ts' 'scripts/**/*.ts'",
```

Comment:
Looks like we're only formatting TS files. Can we format Solidity files too?

For example:

- `prettier:sol` -> uses prettier to format .sol files
- `prettier:ts` -> uses prettier-eslint to format .ts files
- `prettier` -> runs `yarn prettier:ts && yarn prettier:sol`

and then include `yarn prettier` in the Husky pre-commit hook.